server/user/user.controller.js

```diff
@@ -44,9 +44,32 @@ function login({ body }, res) {
     });
 }
 
+function updateProfile({ user, body: { userCredentials } }, res, next) {
+  const { email, firstName, lastName } = userCredentials;
+  return user.update({ email, firstName, lastName })
+    .then(({ profile }) => res.json({ user: profile }))
+    .catch(err => next(err));
```

Comment:
You shouldn't handle `Promise` rejection here because we have global `express` error handler :bowtie: